src/fromTriangles.js

```diff
@@ -0,0 +1,44 @@
+import { inverse } from './inverse'
+import { transform } from './transform'
+import { roundElements } from './roundElements'
+
+/**
+ * Returns a matrix that transforms a triangle t1 into another triangle t2, or throws an exception if it is impossible.
+ * @param t1 [{x: number, y: number}}, {{x: number, y: number}}, {{x: number, y: number}}] array containing the three points for the triangle as a Point array
+ * @param t2 See description for t1.
+ * @returns {{a: number, b: number, c: number, e: number, d: number, f: number}} Affine matrix
+ * @throws Exception if the matrix becomes not invertible
+ */
+export function fromTriangles (t1, t2) {
+  let r1, r2, rx1, ry1, rx2, ry2
+
+  if (Array.isArray(t1) && Array.isArray(t2)) {
```

Comment:
This type checking can be removed. The signature compliance is up to developer that must respect it.
A good feature could be to support Point expressed as array. It's described on the documentation available here https://github.com/chrvadala/transformation-matrix#data-model

Reply:
Thanks for your input! I removed the type check.

Reply:
Added the support for points expressed as arrays.